cmd/slinky-config/main.go

```diff
@@ -222,77 +223,77 @@ func init() {
 		"oracle-config-path",
 		"",
 		"oracle.json",
-		"path to write the oracle config file to. this file is required to run the oracle.",
+		"Path to write the oracle config file to. This file is required to run the oracle.",
 	)
 	rootCmd.Flags().StringVarP(
 		&marketCfgPath,
 		"market-config-path",
 		"",
 		"market.json",
-		"path to write the market config file to. this file is required to run the oracle.",
+		"Path to write the market config file to. This file is required to run the oracle.",
 	)
 	rootCmd.Flags().StringVarP(
 		&chain,
-		"chain-id",
+		"chain",
 		"",
 		"",
-		"chain-id that we expect the oracle to be running on. ex dydx-mainnet-1, dydx-testnet-4. this should only be specified if required by the chain.",
+		"Chain that we expect the oracle to be running on {dydx, \"\"}. This should only be specified if required by the chain.",
 	)
 	rootCmd.Flags().StringVarP(
 		&nodeURL,
 		"node-http-url",
 		"",
 		"",
-		"http endpoint of the cosmos sdk node corresponding to the chain id (typically something like localhost:1317). this should only be specified if required by the chain.",
+		"Http endpoint of the cosmos sdk node corresponding to the chain (typically localhost:1317 or a remote API). This should only be specified if required by the chain.",
 	)
 	rootCmd.Flags().StringVarP(
 		&host,
 		"host",
 		"",
 		"0.0.0.0",
```

Comment:
Are we checking that the default value isn't used if no chain is given here?

Reply:
This variable is only used if dydx chain is specified otherwise it's ignored at the moment.